README.md

```diff
@@ -151,14 +151,14 @@ To use the [Transactional API](https://customer.io/docs/transactional-api), inst
 Use `send_email` referencing your request to send a transactional message. [Learn more about transactional messages and `SendEmailRequest` properties](https://customer.io/docs/transactional-api).
 
 ```python
-from customerio import APIClient
+from customerio import APIClient, SendEmailRequest
 
 client = APIClient("your API key")
 
-request = SendEmailRequest({
-  "to": "person@example.com",
-  "transactional_message_id": "3",
-  "message_data": {
+request = SendEmailRequest(
+  to="person@example.com",
+  transactional_message_id="3",
```

Comment:
if possible please also mention `_from` and `body` in the sample. This `_from` can trip people because in Python library it has to start with an underscore :) 

Here is the Python code you can use as a hint:

```
request = SendEmailRequest(
  to="waqas@customer.io",
  transactional_message_id="8",
  _from="wee@customer.io",
  subject="Hey, how are you?",
  body="Hey this is the body",
  message_data={
    "name": "person",
    "items": [
      {
        "name": "shoes",
        "price": "59.99",
      },
    ]
  },
  identifiers={
    "id": "7575788858585222a",
  }
)
```